### PR TITLE
Add NotchToolKit

### PR DESCRIPTION
**Pull request by AFathi**
Added NotchToolkit framework in the UI list.

## Project URL

https://github.com/AFathi/NotchToolkit

## Description

 An iOS framework that allow developers use the iPhone X notch in creative ways.

## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 / tvOS 10 or later
- [x] Supports Swift 3 or later
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 1 link issue, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issue by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
1268 | [301](https://httpstatuses.com/301) | https://github.com/west0r/SwiftyVK redirects to<br>https://github.com/SwiftyVK/SwiftyVK


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by lfarah** [comment]
Hey @AFathi, good job on the lib and thanks for contributing!

File: README.md [hunk 1]
```diff
@@ -1617,6 +1617,7 @@ Most of these are paid services, some have free tiers.
 * [Pulley](https://github.com/52inc/Pulley) - A library to imitate the iOS 10 Maps UI :large_orange_diamond:
 * [N8iveKit](https://github.com/n8iveapps/N8iveKit) - A set of frameworks making iOS development more fun. :large_orange_diamond:
 * [Panda](https://github.com/wordlessj/Panda) - Create view hierarchies declaratively. :large_orange_diamond:
+* [NotchToolkit](https://github.com/AFathi/NotchToolkit) - A framework for iOS that allow developers use the iPhone X notch in creative ways. 🔶
```
Comment (lfarah) [inline README.md]: Please add your link to the end of the /UI section

Comment (AFathi) [reply README.md]: Hey Lucas, I just added my link to the end of the UI section.

**Comment by dkhamsing** [comment]
Duplicate

**Pull request closed without merging**